### PR TITLE
Blog post draft "Introducing geoextent

### PR DESCRIPTION
**Pull request by YouQam**

### CODE REVIEW

File: _drafts/Introducing-geoextent.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+The output will show the bbox, time interval and crs extracted from file data, as follow: 
+
+```
+{'format': 'text/csv',
+ 'crs': '4326',
+ 'tbox': ['30.09.2018', '30.09.2018'],
+ 'bbox': [4.3175, 51.434444, 6.574722, 53.217222]}
+```
+
+[OpenStreetMap](https://www.openstreetmap.org/export#map=8/52.347/5.446) showing the area of extracted bbox.
```
Comment (nuest) [inline _drafts/Introducing-geoextent.md:20]: That map is configured with a zoom level and latitude longitude coordinates. It does not "show a bounding box". Can you please create a map, that actually shows a line with the bounding box extent?

File: _drafts/Introducing-geoextent.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
+# Introducing geoextent
+
+geoextent is an easy to use library for extracting the geospatial extent from data files with multiple data formats.
+
+Take a look at the [source code on GitHub](https://github.com/o2r-project/geoextent.git), [library on PyPI](https://pypi.org/project/geoextent/) and the [documentation website](https://o2r.info/geoextent/).
+
+Here is a small example how to use `geoextent`.
+```
+geoextent -b -t -input= 'cities_NL.csv'
```
Comment (nuest) [inline _drafts/Introducing-geoextent.md:9]: Add a link where a reader can view the used input file, please.

File: _drafts/Introducing-geoextent.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
+# Introducing geoextent
```
Comment (nuest) [inline _drafts/Introducing-geoextent.md:1]: Please add required metadata into the header, i.e. the title, authors, keywords. See files in `_posts` for examples.

**Comment by nuest** [comment]
Before we publish, we need to release `geoextent` in version `0.1.1`, see https://github.com/o2r-project/geoextent/pull/67

File: _drafts/Introducing-geoextent.md (hunk 1 above)
Comment (YouQam) [reply _drafts/Introducing-geoextent.md:20]: Done.

File: _drafts/Introducing-geoextent.md (hunk 2 above)
Comment (YouQam) [reply _drafts/Introducing-geoextent.md:9]: Done.

File: _drafts/Introducing-geoextent.md (hunk 3 above)
Comment (YouQam) [reply _drafts/Introducing-geoextent.md:1]: Done.

**Pull request merged**